domain/src/main/kotlin/no/nav/su/se/bakover/domain/vedtak/Vedtak.kt

```diff
@@ -355,6 +364,12 @@ fun List<Vedtak>.lagTidslinje(periode: Periode): List<Vedtak.VedtakPåTidslinje>
             periode = it.periode,
             grunnlagsdata = it.behandling.grunnlagsdata,
             vilkårsvurderinger = it.behandling.vilkårsvurderinger,
+            fradrag = when (it) {
+                is Vedtak.Avslag.AvslagBeregning -> TODO()
+                is Vedtak.Avslag.AvslagVilkår -> TODO()
+                is Vedtak.EndringIYtelse -> it.beregning.getFradrag()
+                is Vedtak.IngenEndringIYtelse -> TODO()
```

Comment:
Har disse TODO-ene noen konsekvens på kort sikt? Og er det noen plan for implementasjon av dem?